public/assets/css/components/contributors.css

```diff
@@ -0,0 +1,85 @@
+.contributors {
```

Comment:
This file does not looks compatible with our CSS convention. Please take a look at this comment https://github.com/Vanilla-OS/website/pull/116#discussion_r1290500609

Reply:
Can you please review the file again? I've made some changes